rollup.config.js

```diff
@@ -1,41 +1,33 @@
-import resolve from '@rollup/plugin-node-resolve'
-import commonjs from '@rollup/plugin-commonjs'
-import babel from '@rollup/plugin-babel'
 import preserveShebang from 'rollup-plugin-preserve-shebang'
+import { createPlugins } from 'rollup-plugin-atomic'
+
+const plugins = (isAtom) =>
+  createPlugins(
+    [
+      'js',
+      ['ts', { tsconfig: './src/tsconfig.json' }, true],
+      ['replace', { 'process.env.PACKAGE_DEPS_IS_ATOM': isAtom }, true],
```

Comment:
```suggestion
      ['replace', { 'process.env.PACKAGE_DEPS_IS_ATOM': JSON.stringify(isAtom) }, true],
```

since the value check is now a string, we have to encode it as a string too

Reply:
I already pass these as a string.
https://github.com/steelbrain/package-deps/pull/336/files#diff-6814bf77564b4f1c92f5861e184e28fe217c080a047fefa8b73a728f755ec45cR21

Reply:
That string is parsed raw
`'true'` becomes
```
process.env.X = true
```

What we want is this:

`JSON.stringify('true')` becomes
```
process.env.X = "true"
```

Reply:
I might be wrong here, my knowledge is based off of the Webpack define plugin. I'll double check

Reply:
Confirmed my theory, we need to add `JSON.stringify`

source:
<img width="424" alt="Screen Shot 2021-07-20 at 1 26 05 AM" src="https://user-images.githubusercontent.com/4278113/126222659-459452fd-f8d6-47b4-a3d9-93ab2b56d95c.png">

output 1:
<img width="314" alt="Screen Shot 2021-07-20 at 1 26 21 AM" src="https://user-images.githubusercontent.com/4278113/126222695-046eae61-ef59-45ef-9d69-fce464af2be0.png">

Output 2:
<img width="184" alt="Screen Shot 2021-07-20 at 1 26 31 AM" src="https://user-images.githubusercontent.com/4278113/126222719-19a062c0-dbc7-4936-8115-19a2063d9880.png">

`!1` is minified `false` and `!0` is minified true, literals not string values

Reply:
Yes. Thanks for the fix.